docs/source/users/index.rst

```diff
@@ -8,5 +8,5 @@ Welcome to the documentation for the FitBenchmarking software.
     :caption: Contents:
 
     getting_started
-    examples
+    run_scripts
```

Comment:
This is just a comment but I think "examples" is clearer. Perhaps that just falls under the next step of sorting this out once its copied (e.g. a different PR)

Reply:
I think you're right, so I've just added this PR, while we spot it.